### PR TITLE
Add support to JSX blocks

### LINKED ISSUE #1: Support jsx nodes
How about supporting `jsx` nodes in addition to `html`?

### PR DESCRIPTION
**Pull request by fsmaia**
Fixes #1

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -8,10 +8,23 @@ Hello this is Markdown and it's a fantastic format.
 <!-- This is comment #2 -->
 `;
 
+const jsxBlock = `<b>References:</b> [README](./README.md)<!-- -->, [CONTRIBUTING](./CONTRIBUTING.md)`;
```
Comment (fsmaia) [inline test.js:11]: Note: I have no idea why this block is identified as a JSX block, and it is the exact corner case that led me to this adjustment.

**Comment by alvinometric** [comment]
Hey Fernando,

Thank you so much for this! I'm also confused as to why this is a JSX block 🤷‍♂️
I guess because there was HTML and Markdown on one line it treated it as JSX ? I'm still learning a lot about the unified ecosystem too.

Clear PR all around though, thanks for updating the tests 🙏

**Comment by fsmaia** [comment]
You're welcome!

Take a look: https://github.com/mdx-js/mdx/issues/611
